src/megameklab/com/printing/PrintRecordSheet.java

```diff
@@ -150,16 +150,6 @@ protected final Font getBoldFont(float size) {
     private void assignFonts() {
         typeface = CConfig.getParam(CConfig.RS_FONT, DEFAULT_TYPEFACE);
         Font font = Font.decode(typeface);
-        // If the font is not installed, use system default sans
-        if (null == font) {
-            typeface = Font.SANS_SERIF;
-            font = Font.decode(typeface);
-        }
-        // If that doesn't work, get the default dialog font
-        if (null == font) {
-            font = Font.decode(null);
-            typeface = font.getName();
-        }
```

Comment:
If Font.decode doesn't find the requested typeface it returns the default font anyway.